api/src/modules/userModule.js

```diff
@@ -0,0 +1,35 @@
+const mongoose = require('mongoose');
+const Joi = require('@hapi/joi');
+
+const userSchema = mongoose.Schema({
+    firstName: String,
+    lastName: String,
+    email: String,
+    phone: String,
+    birthday: Date,
+    created: {
+        type: Date,
+        default: Date.now(),
+    },
+    role: String,
+    password: String,
+    password_repeat: String,
```

Comment:
I do not think there is need to store `password_repeat in the data base`. You should check it on the FE side, that the user entered twoo identical password, so no need for server to kow about it